examples/rust/custom_callback/README.md

```diff
@@ -0,0 +1,38 @@
+<!--[metadata]
+title = "Custom Viewer Callback"
+thumbnail = "https://static.rerun.io/custom_callback/1434da408fd59ea1349169784b47d8ffc285022e/480w.png"
+thumbnail_dimensions = [480, 291]
+-->
+
+Example showing how to control an app with the Rerun viewer, by extending the viewer UI.
```

Comment:
the important part here is the communcation back. we have other examples that already talk about viewer extension in `extend_viewer_ui`. So maybe let's stress that a bit more:

```suggestion
Advanced example showing how to control an external application from the Rerun viewer, by extending the viewer UI.
```